repos/fdbt-site/src/utils/apiUtils/validator.ts

```diff
@@ -100,7 +104,7 @@ export const checkPriceIsValid = (inputtedPrice: string | undefined, inputType:
         productPriceError = 'This must be a positive number';
     } else if (!isCurrency(inputtedPrice)) {
         productPriceError = 'This must be a valid price in pounds and pence';
-    } else if (inputType === 'cap' && Number(inputtedPrice) === 0) {
+    } else if (inputType === 'cap' && Number(inputtedPrice) === 0 && !isZeroAllowed) {
         productPriceError = 'Cap prices cannot be zero';
```

Comment:
The error message should be more generic, like the other ones - 'This price must not be zero'.